client/my-sites/customer-home/cards/actions/quick-links/style.scss

```diff
@@ -1,50 +1,29 @@
-.quick-links__boxes {
```

Comment:
One more instance of changes caused by the rename of `quick-links-compact` to `quick-links`, feel free to ignore.